build.gradle

```diff
@@ -0,0 +1,105 @@
+plugins {
+
+    // https://docs.gradle.org/current/userguide/java_library_plugin.html
+    id "java-library"
+
+    // https://docs.gradle.org/current/userguide/distribution_plugin.html
+    id "distribution"
+
+    // https://docs.gradle.org/current/userguide/publishing_maven.html
+    id "maven-publish"
+
+}
+
+group = "io.aiven"
+version = hasProperty("module_version") ? "$project.module_version" : 'unknown'
+
+repositories {
+    jcenter()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+ext {
+    kafkaVersion = "2.2.0"
+    amazonS3Version = "1.11.718"
+    slf4jVersion = "1.7.25"
+}
+
+distributions {
+    main {
+        contents {
+            from jar
+            from configurations.runtimeClasspath
+
+            into("/") {
+                from projectDir
+                include "version.txt", "README*", "LICENSE*", "NOTICE*", "licenses/"
+                include "config/"
+            }
+        }
+    }
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            // Defaults, for clarity
+            groupId = getGroup()
+            artifactId = getName()
+            version = getVersion()
+
+            pom {
+                name = "Aiven Kafka S2 connector"
+                description = "A Kafka connector S3 for syncing between Kafka ands S3."
```

Comment:
Please rephrase this, something like in https://github.com/aiven/aiven-kafka-connect-elasticsearch/blob/master/build.gradle#L82